src/roswire/ros2/node_manager.py

```diff
@@ -1,30 +1,38 @@
 # -*- coding: utf-8 -*-
 __all__ = ('ROS2NodeManager',)
 
-from typing import Iterator, Mapping
+from typing import Iterator
 import typing
+import attr
```

Comment:
Bit of a nitpick: standard library imports should be separated from external imports (see https://www.python.org/dev/peps/pep-0008/#imports)